civicrm.php

```diff
@@ -507,6 +519,9 @@ public function register_hooks_admin() {
     // set page title
     add_filter( 'admin_title', array( $this, 'set_admin_title' ) );
 
+    // print CiviCRM's header
+    add_action('admin_head', array( $this, 'wp_head' ), 50);
```

Comment:
@christianwach this is the 1 *real* change to this file, and I wanted to run it by you. It took me a while to figure out why my call to `addCoreResources` wasn't working in `civicrm.shortcodes.modal.php` and finally realized this wasn't being called. So instead of doing it from there, I thought it would be more developer-friendly to have this action run unconditionally (note removal of conditional action below). If Civi isn't bootstrapped or hasn't added anything to the header, the function does nothing, so seems safe to me.